src/imagery/i.eodag/i.eodag.html

```diff
@@ -0,0 +1,65 @@
+<h2>DESCRIPTION</h2>
+<H3>WARNING: I.EODAG IS STILL UNDER DEVELOPMENT. THIS IS AN EXPERIMENTAL VERSION.</H3>
+
+<em>i.eodag</em> allows to search and download imagery datasets, e.g. Sentinel, Landsat, and MODIS, from a number of different providers.
```

Comment:
@HamedElgizery Please break long lines in the HTML file as per style guide:

https://github.com/OSGeo/grass/blob/main/doc/development/style_guide.md#tool-documentation
-> More notes on markup:
-->  In order to minimize potential git merge conflicts, please break a line at approximately 70-80 chars.

Thanks!